lottie/src/main/java/com/airbnb/lottie/LottieDrawable.java

```diff
@@ -853,7 +853,7 @@ public void playAnimation() {
    *
    * @return The first non-null marker from the list of allowed reduced motion markers, or null if no such marker is found.
    */
-  private Marker getMarkerForAnimationsDisabled() {
+  public Marker getMarkerForAnimationsDisabled() {
```

Comment:
Can you add a `@RestrictTo(Library)` on this one?

Reply:
done